apps/wallet/src/ui/app/staking/validators/ValidatorsCard.tsx

```diff
@@ -40,14 +41,27 @@ export function ValidatorsCard() {
         return getAllStakeSui(delegatedStake);
     }, [delegatedStake]);
 
+    const { data: inActiveStakingPoolID } = useGetInactiveStakingPoolIds(
+        system?.inactivePoolsId
+    );
+
     const delegations = useMemo(() => {
         return delegatedStake?.flatMap((delegation) => {
             return delegation.stakes.map((d) => ({
                 ...d,
+                // flag any inactive validator for the stakeSui object
+                inactiveValidator: inActiveStakingPoolID?.includes(
```

Comment:
I'm a little confused how this works out. Does this fetch _all_ of the inactive staking pools and check for inclusion of this specific staking pool? If so that's probably not going to scale and we'll need another solution.